src/components/dashboard/sidebar.tsx

```diff
@@ -1,19 +1,28 @@
 import { LanguageSwitch } from '@/components/dashboard/language-switch'
-import { SidebarItem } from '@/components/dashboard/sidebar-item'
-import { EXTRA_ROUTES, MAIN_ROUTES } from '@/components/dashboard/utils'
 import { Logo } from '@/components/logo'
-import { Flex, Spacer, Stack, StackProps, Text } from '@chakra-ui/react'
+import { Flex, Stack, StackProps, Text, Spacer } from '@chakra-ui/react'
+import { MAIN_ROUTES, EXTRA_ROUTES } from '@/data/SidebarData'
 import React from 'react'
+import { SidebarItem } from './sidebar-item'
 
 export const Sidebar = (props: StackProps) => {
   return (
-    <Stack w="full" h="full" py="4" spacing="10" {...props}>
+    <Stack
+      w="full"
+      h="full"
+      py="4"
+      spacing="10"
+      {...props}
+      borderRight="solid 1px"
+      borderColor="gray.200"
+    >
       <Flex
         gap="2"
         align="center"
         px="5"
+        py="3"
         borderBottom="solid 1px"
-        borderColor={{ base: 'whiteAlpha.200', md: 'transparent' }}
+        borderColor={{ base: 'gray.200', md: 'transparent' }}
```

Comment:
I've only developed for dark mode. If you see any color that looks weird in light mode, or is non existent in light mode, don't change them, instead create a semantic token, and add the light mode equivalent.
E.g.:
```suggestion
        borderColor={{ base: 'cardBorder', md: 'transparent' }}
```

Reply:
Okay then.